src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -1506,11 +1506,40 @@
         "4.3.0",
         "4.4.0",
         "4.5.0",
-        "4.6.0"
+        "4.6.0",
+        "4.7.0"
       ],
       "BaselineVersion": "5.0.0",
       "InboxOn": {}
     },
+    "runtime.linux-arm.runtime.native.System.IO.Ports": {
+      "StableVersions": [
+        "4.7.0",
+        "5.0.0"
+      ],
+      "InboxOn": {}
+    },
+    "runtime.linux-arm64.runtime.native.System.IO.Ports": {
+      "StableVersions": [
+        "4.7.0",
+        "5.0.0"
+      ],
+      "InboxOn": {}
+    },
+    "runtime.linux-x64.runtime.native.System.IO.Ports": {
+      "StableVersions": [
+        "4.7.0",
+        "5.0.0"
+      ],
+      "InboxOn": {}
+    },
+    "runtime.osx-x64.runtime.native.System.IO.Ports": {
```

Comment:
What happens if we add new rids, do we have to be adding more entries here? I don't know if there is a better way long term. For 5.0.0 fix, it seems good as is, but maybe we can come with a better way long term.

Reply:
In 6.0 the package index will go away.